planning/behavior_velocity_planner/autoware_behavior_velocity_planner/src/node.hpp

```diff
@@ -95,11 +93,16 @@ class BehaviorVelocityPlannerNode : public rclcpp::Node
   autoware::universe_utils::InterProcessPollingSubscriber<nav_msgs::msg::OccupancyGrid>
     sub_occupancy_grid_{this, "~/input/occupancy_grid"};
 
+  autoware::universe_utils::InterProcessPollingSubscriber<LaneletMapBin> sub_lanelet_map_{
```

Comment:
```suggestion
  autoware::universe_utils::InterProcessPollingSubscriber<LaneletMapBin, universe_utils::polling_policy::Newest> sub_lanelet_map_{
```

When I use Newest policy, it seems that this node can't subscribe map data.